tests/Nelmio/Alice/Fixtures/Parser/Methods/PhpTest.php

```diff
@@ -110,4 +110,31 @@ public function testIncludeFiles()
         ];
         $this->assertEquals($expectedData, $data);
     }
+
+    public function testIncludeFilesOrder()
+    {
+        $data = $this->parser->parse(__DIR__.'/../../../support/fixtures/include_order.php');
+
+        $expectedData = [
+            'Foo' =>
+                [
+                    'foo' =>
+                        [
+                            'id' => 1,
+                            'text' => '<word()>',
+                        ],
+                ],
+            'Bar' =>
+                [
+                    'bar' =>
+                        [
+                            'id' => 1,
+                            'text' => '<@foo->text>'
+                        ],
+                ],
+        ];
+
+        $this->assertEquals($expectedData, $data);
+        $this->assertTrue($expectedData === $data);
```

Comment:
I'll try to explain it better.

assertEquals() will print diff-style message when values of $expectedData and $data are not the same, helping you to find out why the test has failed.

Here is an example of what you'll get if for some reason the test subject is different from what expected:

```
1) Nelmio\Alice\Fixtures\Parser\Methods\YamlTest::testIncludeFilesOrder
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
 Array (
     'Foo' => Array (
         'foo' => Array (
-            'id' => 2
+            'id' => 1
             'text' => '<word()>'
         )
     )
     'Bar' => Array (...)
 )
```
As assertEquals() will use == operator to compare the arrays, it will not check if the order is the same.

assertTrue() instead, will ensure that sorting is preserved, but unlike assertEquals(), it will not print a diff-style message to help you.

Another way to write it with just one assert, could be:
```
$this->assertSame($expectedData, $data)
```

This will print a slightly different message from those printed by assertEquals() in case of failure. Here is an example:

```
1) Nelmio\Alice\Fixtures\Parser\Methods\YamlTest::testIncludeFilesOrder
Failed asserting that Array &0 (
    'Foo' => Array &1 (
        'foo' => Array &2 (
            'id' => 1
            'text' => '<word()>'
        )
    )
    'Bar' => Array &3 (
        'bar' => Array &4 (
            'id' => 1
            'text' => '<@foo->text>'
        )
    )
) is identical to Array &0 (
    'Foo' => Array &1 (
        'foo' => Array &2 (
            'id' => 2
            'text' => '<word()>'
        )
    )
    'Bar' => Array &3 (
        'bar' => Array &4 (
            'id' => 1
            'text' => '<@foo->text>'
        )
    )
).
```